### ISSUE TITLE
Grub does not detect other operating systems on installation

### ISSUE THREAD

**Issue by olivaresa** [aoxlinux/aoxlinux#3]
At the end of the installation on a PC with other Operating Systems the Grub does not completely update the registers for booting, it only adds entries for Aox Linux and ignores the other systems.

**Comment by olivaresa** [comment]
It is solved by executing the following command in a console and then reboot the system:

```
sudo bash -c "sed -i 's/#GRUB_DISABLE_OS_PROBER=false/GRUB_DISABLE_OS_PROBER=false/g' /etc/default/grub; grub-mkconfig -o /boot/grub/grub.cfg"
```

**Comment by olivaresa** [comment]
Fixed in v0.2-beta
